src/Command/CommandBase.php

```diff
@@ -1155,4 +1155,98 @@ protected function convertEnvironmentAliasToUuid(InputInterface $input): void {
     }
   }
 
+  /**
+   * @param $cloud_environment
+   *
+   * @return string
+   */
+  public static function getSiteGroupFromSshUrl($cloud_environment): string {
+    $ssh_url_parts = explode('.', $cloud_environment->sshUrl);
+    $sitegroup = reset($ssh_url_parts);
+
+    return $sitegroup;
+  }
+
+  /**
+   * @param $cloud_environment
+   *
+   * @return bool
+   */
+  protected function isAcsfEnv($cloud_environment): bool {
```

Comment:
These functions are not specific to pushing/pulling files, and could be useful more broadly, so I moved them here.